### ISSUE TITLE
Clarify the need to setup Wordpress locally

### ISSUE THREAD

**Issue by mzgol** [jquery/contribute.jquery.org#124]
I wanted to fix one thing on jquery.com and I didn't do anything there for a long time so I decided to try with freshly cloned repos with the new Vagrant setup. I started from:
https://github.com/jquery/jquery.com/blob/master/README.md
which redirects to:
https://github.com/jquery/jquery-wp-content
(basically just "install Virtualbox & Vagrant & run `vagrant up`" - simple) and:
http://contribute.jquery.org/web-sites/#workflow
However, it didn't work for me, I was getting `wordpress-validate` errors about and unknown error that may mean Wordpress or the database is not set properly. Now, instructions at the contribute url say:
"Edit `config.json` to use the URL, username, and password for your local WordPress network."
which may suggest you first need to set up the Wordpress instance... but with having most of the setup handled by Vagrant, I thought that's handled as well.

It would save me (& maybe more people) some time if a step saying explicitly about going to vagrant.jquery.com & setting it up first was introduced.

**Comment by scottgonzalez** [comment]
> basically just "install Virtualbox & Vagrant & run vagrant up" - simple

Well, that and:

> Complete the installation process by pointing your browser at http://vagrant.jquery.com/. Feel free to use any username/password/email combination that you choose, though keep note of the username and password as you will need them to deploy to your local VM.

This comes directly from the [setup instructions](https://github.com/jquery/jquery-wp-content#setup).

**Comment by arthurvr** [comment]
Sounds like there are definitely some things we can clarify on the site. Assigning this one to me, I'll work on it.

> It would save me (& maybe more people) some time if a step saying explicitly about going to http://vagrant.jquery.com & setting it up first was introduced.

:+1:

**Comment by mzgol** [comment]
> Complete the installation process by pointing your browser at http://vagrant.jquery.com/

OK, apparently I'm blind. I'll leave it to you to decide if it should be highlighted more visibly or if current instructions are fine.

**Comment by scottgonzalez** [comment]
I think the README can certainly use some better formatting/editing so that there's not a massive block of expected output before the next (important) step.

**Comment by mzgol** [comment]
> so that there's not a massive block of expected output before the next (important) step.

Yeah, that is most likely what made me jump too far into the list.
